### PR TITLE
person-renderer: Add ellipses-truncation

### PR DESCRIPTION
**Pull request by skye2k2**
Set 100% width on each parent container, so that text-overflow: ellipsis can function.
Specify min-height and min-width for fs-icons, so that they do not scale smaller because of the 100% widths or when the base font size changes.

Reviewed-by: Drew McMurry

### CODE REVIEW

**Comment by skye2k2** [comment]
The proposed code change works better than our current implementation, which purely truncates content at container boundaries, and was submitted as a courtesy to the component maintaining team instead of opening an issue to complain. Both the person card and person summary extended header depend on this functionality. Alternative solutions are welcome, but we need "good enough" now.

File: fs-person/fs-person.html [hunk 1]
```diff
@@ -27,6 +28,11 @@
       :host([inline]) .fs-person-vitals__name {
         margin-right: 15px;
       }
+      .fs-person-vitals,
+      .fs-person-vitals__name-block {
+        width: 98%;
+        width: calc(100% - 1.143rem);
```
Comment (Norman-Amy) [inline fs-person/fs-person.html]: I mean does the gender icon disappearing hurt your changes?

Comment (skye2k2) [inline fs-person/fs-person.html]: We end up prematurely truncating text ~16px early, which is still better than being cut by the parent container. Hence the future optimization, or someone smarter than me making `.fs-person-vitals, .fs-person-vitals__name-block` have 100% width when we are not showing the gender icon.

Comment (Norman-Amy) [inline fs-person/fs-person.html]: Could we add `:host([inline])` in front of those selectors and then have their default be 100% width in another block? That would give the name a bit more room.

**Comment by jshcrowthe** [comment]
LGTM 👍  I addressed the calc issue and left a comment in the code with a link to why.

File: fs-person/fs-person.html (hunk 1 above)
Comment (jshcrowthe) [inline fs-person/fs-person.html]: Flexbox addresses the dynamic width, the truncation issue was related to flexbox's handling of text-width.

See https://css-tricks.com/flexbox-truncated-text

**Pull request merged**